$OpenBSD: patch-unix_installManPage,v 1.4 2013/11/04 16:30:46 stu Exp $

Don't misrename manpages. Tk bug 3604346.
Put macros after .TH to prevent them from being redefined. Tcl bug 426679ef7d.

--- unix/installManPage.orig	Wed Dec 19 10:40:38 2012
+++ unix/installManPage	Tue Jul  9 11:49:35 2013
@@ -71,7 +71,7 @@ Names=`sed -n '
 	s/ \\\-.*//
 #                               Convert all non-space non-alphanum sequences
 #                               to single underscores.
-	s/[^ A-Za-z0-9][^ A-Za-z0-9]*/_/g
+#	s/[^ A-Za-z0-9][^ A-Za-z0-9]*/_/g
 #                               print the result and exit
 	p;q
     }' $ManPage`
@@ -104,7 +104,7 @@ for Target in $Names; do
     rm -f $Dir/$Target $Dir/$Target.*
     if test -z "$First" ; then
 	First=$Target
-	sed -e "/man\.macros/r $SrcDir/man.macros" -e "/man\.macros/d" \
+	sed -e "/man\.macros/d" -e "/\.TH/r $SrcDir/man.macros" \
 	    $ManPage > $Dir/$First
 	chmod 444 $Dir/$First
 	$Gzip $Dir/$First
